README.md

```diff
@@ -9,16 +9,20 @@ OTA devices communicate with a device-gateway using SSL client
 authentication. This means the public key of each device is known. By
 default, devices will have Ellipitcal Curve keys. Using a technique
 known as [ECIES](https://cryptopp.com/wiki/Elliptic_Curve_Integrated_Encryption_Scheme)
-a the configuration values can be encrypted client-side and sent to
+the configuration values can be encrypted client-side and sent to
 the device-gateway so that it has no knowledge of a device's configuration
 values. The device can then pull down the encrypted configuration and
 use its private key to decrypt.
 
-The encrypted file is stored to a persistent location on disk. At boot,
+The encrypted file is stored in a persistent location on disk. At boot,
 a fioconfig can extract this data to tmpfs (/var/run/secrets) so that
 they are only available at runtime.
 
 
 ## How to build
 `make bin/fioconfig-linux-amd64`
+<br />
+
 `make test`
+
```

Comment:
I appreciate grammar corrections to the readme.
Could you make them a separate commit/PR, so they are less distracting a topic of this change?

Reply:
Ok